.changeset/afraid-rings-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-color": patch
```

Comment:
**nit:** This is a breaking change for code that relies on the old behavior. As such I think this likely needs to be a `major` change, though I could be persuaded down to `minor` since it is unlikely that there is code relying on this.